scenes.go

```diff
@@ -120,19 +136,23 @@ func scene2() (*config, error) {
 	}
 
 	return &config{
-		Start:      startInput,
-		Goal:       spatialmath.NewPose(goalPt, startPose.Orientation()),
-		RobotFrame: model,
-		WorldState: worldState,
+		Start:            startInput,
+		Goal:             spatialmath.NewPose(goalPt, startPose.Orientation()),
+		RobotFrame:       model,
+		EndEffectorFrame: nil,
+		WorldState:       worldState,
 	}, nil
 }
 
-// scene3: setup a UR5 to move to the other side of an obstacle that obstructs the direct path
 func scene3() (*config, error) {
 	model, _ := universalrobots.Model("arm")
 	startInput := referenceframe.FloatsToInputs([]float64{0, 0, 0, 0, 0, 0})
 	startPose, _ := model.Transform(startInput)
+
+	// Goal pose
 	goalPt := r3.Vector{X: -400, Y: 350, Z: 0}
```

Comment:
`goalPt` could be in terms of `startPose.Point()`.
If we want to execute all scenes with all major robotic arm models this could be a way to figure out common reachable points..

Reply:
Could be, but I think that's best left to the scene implementation. If you recall we got all of the last demo poses in as unique instances referenced to the `world`, not as functions of the start poses, so the person migrating that application (in this case, me) into this repo would need to do some math, which could be a source of error that would not be immediately obvious until you redid the calculations.